### PR TITLE
Skips the warning for NVIDIA driver on sle_sp3

### PR DESCRIPTION
**Pull request by b10n1k**
- Related ticket: https://progress.opensuse.org/issues/70141
- Needles: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/-/merge_requests/1421
- [Verification run]( https://openqa.suse.de/tests/overview?build=b10n1k%2Fos-autoinst-distri-opensuse%2310884&distri=sle&version=15-SP3) 
[autoyast_home_encrypted](http://aquarius.suse.cz/tests/3239#step/installation/4)
[we-module](http://aquarius.suse.cz/tests/3238#step/scc_registration/22)

### CODE REVIEW

**Comment by b10n1k** [comment]
- [x] VR
- [x] Commit message describes approach and decision
- [x] dead code

File: lib/workarounds/bugzilla_issues.pm [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+use testapi;
+use utils;
+use warnings;
+use strict;
+use Exporter;
+our @EXPORT_OK = qw(
+  workaround_1175335
+);
+
+sub workaround_1175335 {
```
Comment (rwx788) [inline lib/workarounds/bugzilla_issues.pm:28]: So as per our discussion, I would put this workaround in `lib/registration.pm` as of now. But we still might think of having separate file for it. It's also not that important, as we will have to revert this change, once bug fix is there.

More important thing not to miss is to add `soft_failure` as we don't add any visible reference to the bug.

File: tests/autoyast/installation.pm [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2015-2019 SUSE Linux GmbH
+# Copyright © 2015-2020 SUSE LLC
```
Comment (rwx788) [inline tests/autoyast/installation.pm:1]: Please, put this change and change to `tests/installation/scc_registration.pm` to separate commit, so we don't loose it when revert the workaround.

File: tests/installation/scc_registration.pm [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2014-2019 SUSE LLC
+# Copyright © 2014-2019 SUSE LLC
```
Comment (rwx788) [inline tests/installation/scc_registration.pm:1]: I guess we can put 2020 here, as long as you decided to make it better ;)

File: tests/autoyast/installation.pm (hunk 2 above)
Comment (b10n1k) [reply tests/autoyast/installation.pm:1]: done in separate commit

File: lib/workarounds/bugzilla_issues.pm (hunk 1 above)
Comment (b10n1k) [reply lib/workarounds/bugzilla_issues.pm]: Ok. i tried to think something but i think it is not worth the time to decide one or the other for now. Maybe we can discuss it outside of a PR. i simplified it
